extensions/blocks/simple-payments/simple-payments.php

```diff
@@ -45,6 +45,19 @@ function render_block( $attr, $content ) {
 		return $content;
 	}
 
+	$form_errors_markup = '';
+	if ( isset( $attr['price'] ) ) {
+		$price = intval( $attr['price'] );
+		if ( $price <= 0 ) {
+			// translators: %s: product price.
+			$form_errors_markup .= '<li>' . sprintf( __( '<strong>Error</strong>: Invalid price value: %s', 'jetpack' ), $price ) . '</li>';
+		}
+	}
+
+	if ( ! empty( $form_errors_markup ) && ! is_user_logged_in() && ! is_admin() ) {
+		return '';
+	}
```

Comment:
Maybe we could take advantage of `Jetpack_Gutenberg::notice()` to display those errors?
https://github.com/Automattic/jetpack/blob/ebec0a8fb35ffac4766cce36646ebf01c5d56063/class.jetpack-gutenberg.php#L960-L971